.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: CI
+on: [ push, pull_request ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```
Fix the mainline release to keep a consistent behavior over time, ease contributor reproductability, and remove warnings about upcoming upgrade. WDYT?

---

OK